utils/fake_data.py

```diff
@@ -120,24 +128,26 @@ def __init__(self):
             DataType.CITY: self.fake.city,
             DataType.STATE: self.fake.state_abbr,
             DataType.POSTAL_CODE: self.fake.safe_postal_code,
+            DataType.POSTAL_CODE_VARIANT: self.fake.safe_postal_code,
             DataType.COUNTRY: self.fake.country,
             DataType.DATE: self.fake.date,
             DataType.DATE_TIME: self.fake.iso_date_time,
             DataType.TIME_MILITARY: self.fake.time_military,
             DataType.SENTENCE: self.fake.sentence,
             DataType.BOOLEAN: self.fake.boolean,
-            DataType.INTEGER: self.fake.random_number,
+            DataType.INTEGER: self.fake.random_int,
             DataType.UUID: self.fake.safe_uuid,
+            DataType.TAC: self.fake.tac,
         }
 
     def get_random_choice(self, choices):
         """ Given a list of random choices, returns one of them. """
         return self.fake.random_element(choices)
 
-    def get_fake_data_for_type(self, data_type):
+    def get_fake_data_for_type(self, data_type, params=None):
         """ Given a specific data type, returns faker data for that type (if a mapping exists). """
         try:
-            return self.data_types[data_type]()
+            return self.data_types[data_type](*params)
```

Comment:
nice, this is a good addition. `**params` instead might be a little safer, but I think this is probably fine

Reply:
So **params would handle the case where we wanted a Dict's items converted to params?  I think that's what I understood is the difference from the docs.